iis/Makefile.win

```diff
@@ -101,12 +103,21 @@ $(OBJS2): ..\standalone\$*.c
 .cpp.obj:
         $(CC) $(CFLAGS) -c $< -Fo$@
 
-$(DLL): $(OBJS1) $(OBJS2) $(OBJS3) $(OBJS4)
-        $(LINK) $(LDFLAGS) $(OBJS1) $(OBJS2) $(OBJS3) $(OBJS4) $(LIBS)
+$(DLL): $(OBJS1) $(OBJS2) $(OBJS3) $(OBJS4) $(OBJS5)
+        $(LINK) $(LDFLAGS) $(OBJS1) $(OBJS2) $(OBJS3) $(OBJS4) $(OBJS5) $(LIBS)
         IF EXIST $(DLL).manifest $(MT) -manifest $(DLL).manifest -outputresource:$(DLL);#1
 
+$(OBJS5): ModSecurityIISMessage.rc
+        rc /r ModSecurityIISMessage.rc
+
+ModSecurityIISMessage.rc: ModSecurityIISMessage.mc
+        mc -U ModSecurityIISMessage.mc
+
 clean:
 	del /f *.obj
 	del /f *.dll
 	del /f *.exp
 	del /f *.lib
+        del /f *.rc

```

Comment:
Are you sure the indentation level is good here (and below lines)?

May be these are the good ones, but then could you align the above ones (in lines 117-120)?

(Perhaps those are not TAB's but spaces - Makefile prefers TAB's, but may be on Windows this works too.)